tests/integration/test_logcollector/test_macos/test_macos_multiline_values.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+
+import wazuh_testing.logcollector as logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.remote import check_agent_received_message
+# Marks
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=0)]
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_macos_format_basic.yaml')
+
+configurations = load_wazuh_configurations(configurations_path, __name__)
+
+macos_log_messages = [
+    {
+        'command': 'logger',
+        'message': "Here is a multiline log. Line 0 \nLine 1. \nLast line.",
+    }
+]
+
+
+# fixtures
+@pytest.fixture(scope="module", params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def get_connection_configuration():
+    """Get configurations from the module."""
+    return logcollector.DEFAULT_AUTHD_REMOTED_SIMULATOR_CONFIGURATION
+
+
+@pytest.mark.parametrize('macos_message', macos_log_messages)
+def test_macos_multiline_values(get_configuration, configure_environment, get_connection_configuration,
+                                init_authd_remote_simulator, macos_message, restart_logcollector):
+
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over. That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
In case of the need of all Wazuh daemon, the test should disable every module not necessary for the testing in ossec.conf.

The reasons for this change are:

- Faster testing
- More encapsulated tests

Also, the use of simulators is not allowed, it is necessary to implement this test without using simulators.